### ISSUE TITLE
Add configuration option to only run swift format when a .swiftformat file exists

### ISSUE THREAD

**Issue by adam-fowler** [vknabel/vscode-swiftformat#20]
I'd like to enable this extension universally, but it means if I work on a project which isn't using swift format it applies the default formatting to it when I'd rather it did nothing. The existence of a `.swiftformat` file is a pretty good indicator that swift format is being used. 

Can we add a configuration option to only run swift format is a `.swiftformat` file exists in the base of the project

**Comment by vknabel** [comment]
Thanks for the great suggestion!
I'll try to implement this within the next two weeks.

**Comment by vknabel** [comment]
Has been released as [1.4.0](https://github.com/vknabel/vscode-swiftformat/releases/tag/1.4.0).

Within the next few minutes, I will also publish updates for [vknabel/vscode-apple-swift-format](https://github.com/vknabel/vscode-apple-swift-format) and [vknabel/vscode-swiftlint](https://github.com/vknabel/vscode-swiftlint) which implement the same feature.

```json
{
  "swiftformat.onlyEnableWithConfig": true,
  "apple-swift-format.onlyEnableWithConfig": true,
  "swiftlint.onlyEnableWithConfig": true
}
```
